File: frontend/src/api/utils/notifications-endpoints.ts
export const NotificationEndpoints = {
  Admin: {
    Get: 'api/notifications/admin',
    GetUnreadCount : 'api/notifications/admin/count',
    MarkAllRead: 'api/notifications/admin',
    MarkSingleRead: (notificationId: number) => `api/notifications/admin/${notificationId}`,
  },
} as const;

File: backend/Booking/src/Modules/Booking.Modules.Notifications/Features/NotificationsEndpoints.cs
namespace Booking.Modules.Notifications.Features;

public static class NotificationsEndpoints
{
    public static class Admin
    {
        public const string get = "api/notifications/admin";
        public const string markAllRead = "api/notifications/admin";
        public const string markSignleRead = $"api/notifications/admin/{{notificationId}}";
    }
}